src/lib/components/composed/search/Filters.svelte

```diff
@@ -341,8 +342,14 @@
     if(tmpFilter.type == 'select') {
       tmpFilter.values = newValue
     } else if('mode' in tmpFilter && tmpFilter.mode == 'between') {
-      tmpFilter.to = newValue.to
-      tmpFilter.from = newValue.from
+      if(tmpFilter.type == 'date'){
+        tmpFilter.to = DateTime.fromJSDate(newValue.to).setZone('UTC', { keepLocalTime: true }).endOf('day').toJSDate()
```

Comment:
Non mi quadra tanto che a frontend dobbiamo settare la zona a utc mantenendo il time locale. Sei sicuro che devi fare così?

Secondo me basterebbe fare DateTime.fromJSDate(newValue.to).setLocale('it-IT').endOf('day').